recipes/r-stochvol/meta.yaml

```
@@ -0,0 +1,95 @@
+{% set version = '3.2.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-stochvol
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/stochvol_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/stochvol/stochvol_{{ version }}.tar.gz
+  sha256: 5c149912cbb774c0baccc62ee6b78fcdae378595d143a5a17c7bb56f89cd28b3
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
```

Either skip Windows (`skip: true  # [win]`) or whitelist DLLs (as per https://github.com/conda/conda-build/pull/4786):

```suggestion
  missing_dso_whitelist:
    - '*/R.dll'        # [win]
    - '*/Rblas.dll'    # [win]
    - '*/Rlapack.dll'  # [win]

```